Sources/XCTest/XCAbstractTest.swift

```diff
@@ -33,4 +64,8 @@ public class XCTest {
     /// Teardown method called after the invocation of each test method in the
     /// class.
     public func tearDown() {}
+
+    // FIXME: This initializer is required due to a Swift compiler bug on Linux.
+    //        It should be removed once the bug is fixed.
+    public init() {}
```

Comment:
// Not all expectations were fulfilled. Append a failure
              // to the array of expectation-based failures.
              let descriptions = unfulfilledDescriptions.joined(separator: ", ")
 -            let failure = XCTFailure(
 -                message: "Exceeded timeout of \(timeout) seconds, with unfulfilled expectations: \(descriptions)",
 -                failureDescription: "Asynchronous wait failed",
 -                expected: true,
 -                file: file,
 -                line: line)
 -            if let failureHandler = XCTFailureHandler {
 -                failureHandler(failure)
 -            }
 +            recordFailureWithDescription(
 +                "Asynchronous wait failed - Exceeded timeout of \(timeout) seconds, with unfulfilled expectations: \(descriptions)",
 +                inFile: String(file),